src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java

```diff
@@ -0,0 +1,35 @@
+package no.nav.veilarboppfolging.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+@Repository
+@Slf4j
+public class KafkaProducerRepository {
```

Comment:
Kan navnet bli litt forvirrende siden det finnes følgende klasse i common,
`no.nav.common.kafka.producer.feilhandtering.KafkaProducerRepository`? Forslag til annet navn kunne f.eks. vært `KafkaProducerMetricRepository`, eller bare `KafkaMetricRepository` hvis det blir aktuelt med metrikker for consumer også?

Reply:
Yep, I will fix